homeassistant/components/netgear/sensor.py

```diff
@@ -37,23 +42,221 @@
         key="ssid",
         name="ssid",
         entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:wifi-marker",
     ),
     "conn_ap_mac": SensorEntityDescription(
         key="conn_ap_mac",
         name="access point mac",
         entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:router-network",
     ),
 }
 
 
+@dataclass
+class NetgearSensorEntityDescription(SensorEntityDescription):
+    """Class describing Netgear sensor entities."""
+
+    value: Callable = None
```

Comment:
If this attribute is required create a mixin dataclass for it. Search for mixin in sensor platforms for examples.

Reply:
```suggestion
    value: Callable = lambda data: data
```

Reply:
I set a more proper default now. I hope this is okay.